src/System Application/Test/Rest Client/src/RestClientTests.Codeunit.al

```diff
@@ -417,17 +482,160 @@ codeunit 134971 "Rest Client Tests"
         HttpResponseMessage := RestClient.Get('https://httpbin.org/basic-auth/user01/Password123');
 
         // [THEN] The response contains the expected data
-        Assert.AreEqual(HttpResponseMessage.GetHttpStatusCode(), 200, 'The response status code should be 200');
+        Assert.AreEqual(200, HttpResponseMessage.GetHttpStatusCode(), 'The response status code should be 200');
+        JsonObject := HttpResponseMessage.GetContent().AsJson().AsObject();
+        Assert.AreEqual(true, GetJsonToken(JsonObject, 'authenticated').AsValue().AsBoolean(), 'The response should contain the expected data');
+    end;
+
+    [Test]
+    procedure TestResponseWithCookies()
+    var
+        RestClient: Codeunit "Rest Client";
+        HttpResponseMessage: Codeunit "Http Response Message";
+    begin
+        // [SCENARIO] Test GET request with cookies
+
+        // [GIVEN] An initialized Rest Client
+        HttpClientHandler.Initialize();
+        RestClient.Initialize(HttpClientHandler);
+
+        // [WHEN] The Get method is called
+        HttpResponseMessage := RestClient.Get('https://postman-echo.com/get');
```

Comment:
We can't have this. MS blocked Postman due to security concerns, we cannot have tests calling Postman. Actually we cannot have tests calling an external service at all, I'll comment on that later

Reply:
Calling this API most likely would not be a risk but we shouldn't take a change

Reply:
The other tests also call an external service: httpbin.org which has been accepted before.
Testing this module is close to impossible without actually calling an external service. Rest assured I did the tests. How should we work around this?

Reply:
The way I see it, there are two options:
1) Change postman call to something else and disable the test to be run during test automation for now. The test can be run manually when code is churned.
2) Wait for the mocking capabilities to come along (should be soon) and refactor tests to uptake those.

I'd vote for 1, with planned work to do 2 when the time comes. I stand by my "approved" verdict.

Reply:
The challenge is to find another service I can use for testing cookies. Tried httpbin.org first, but it didn't work because of an automatic redirect loosing cookies. It was planning to test this specific scenario and maybe report it as a bug.
To avoid further delays, I searched for another service and found postman-echo.com. I'm open to alternatives. Or should we build our own testing service, e.g., an Azure function, for testing purposes?

Do I need to do anything in the code to disable the test during test automation?

Reply:
I've looked into the DisabledTests folder and found a recent PR that removed the Rest Client tests from the disabled list.
https://github.com/microsoft/BCApps/pull/2142

Do you want this to be added again? Or only for the new tests?

Reply:
Let's only disable the tests we must - not the entire codeunit.

Reply:
I've added the tests for cookies to the disabled tests and kept the already existing list.